packages/runtime/src/widgets/Form/FormItem.tsx

```diff
@@ -20,35 +21,43 @@ export const EnsembleFormItem: React.FC<EnsembleFormItemProps<unknown>> = (
   const formInstance = AntForm.useFormInstance();
   const requiredRule = { required: Boolean(values?.required) };
 
+  const fieldLabel = useMemo(() => {
+    return values?.label ? (
+      <label
+        htmlFor={values.id ?? values.label}
+        style={{
+          ...values.labelStyle,
+        }}
+        title={values.label}
+      >
+        {isString(values.label)
+          ? values.label
+          : EnsembleRuntime.render([unwrapWidget(values.label)])}
+      </label>
+    ) : null;
```

Comment:
Use if and then return early for better readability whenever possible. Avoid ternarnies.